.sbtopts

```diff
@@ -1 +1 @@
--J-XX:MaxMetaspaceSize=3G
\ No newline at end of file
+-XX:MaxMetaspaceSize=3G
```

Comment:
This one was weird it didn't work locally for me, and it looks like a strange option, was the `-J` intentional?